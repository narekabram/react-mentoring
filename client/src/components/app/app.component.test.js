import React from 'react';
import { shallow } from 'enzyme';
import App from './app';

describe('App', () => {
    it('Adds App snapshot testing', () => {
        const component = shallow(<App />);

        expect(component).toMatchSnapshot();
    });
});