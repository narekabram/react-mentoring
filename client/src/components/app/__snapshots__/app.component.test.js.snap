// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Adds App snapshot testing 1`] = `ShallowWrapper {}`;
